File: .ci/doc/templates/print-result-array.tpl.java
try {
  AbstractProtocol protocol;
  if (System.getenv("SNIPPET_PROTOCOL") != null && System.getenv("SNIPPET_PROTOCOL").equals("http")) {
    protocol = new Http("kuzzle");
  } else {
    protocol = new WebSocket("kuzzle");
  }
  kuzzle = new Kuzzle(protocol);
  kuzzle.connect();
  [snippet-code]
  for (Object o : result.get("successes")) {
    System.out.println(o);
  }
} catch (Exception e) {
  e.printStackTrace();
} finally {
  if (kuzzle != null) {
    kuzzle.disconnect();
  }
}

File: .ci/doc/templates/print-result-array.tpl.kt
val protocol: AbstractProtocol;
if (System.getenv("SNIPPET_PROTOCOL") != null && System.getenv("SNIPPET_PROTOCOL") == "http") {
  protocol = Http("kuzzle")
} else {
  protocol = WebSocket("kuzzle")
}
val kuzzle = Kuzzle(protocol).apply {
  connect()
}
try {
  [snippet-code]
  result["successes"]?.map {
    for (item: Any? in it as KuzzleMap) {
      println(item.toString())
    }
  }
  println(result.toString())
} catch (e: Exception) {
  e.printStackTrace()
} finally {
  kuzzle.disconnect()
}